spec/controllers/exercises_controller_spec.rb

```diff
@@ -236,8 +236,8 @@
     expect_template(:statistics)
   end
 
-  describe 'GET #statistics for external users' do
-    let(:perform_request) { get :statistics, params: params }
+  describe 'GET #external_user_statistics for external users' do
```

Comment:
This is duplicating the aspect of "external users" now.

This would be sufficient for me:
```suggestion
  describe 'GET #external_user_statistics' do
```

Reply:
That's true, I'll change that! Thanks! :)